docs/operate/get-started/setup-micro/_index.md

```diff
@@ -0,0 +1,404 @@
+
+   ```sh { class="command-line" data-prompt="$"}
+   make flash-esp32-bin
+   ```
+
+   When prompted, select the serial port that your ESP32 is connected to through a data cable.
+
+   If the flash is successful, you will retain a serial connection to the board until you press `Ctrl-C`.
+   While the serial connection is live, you can also restart the currently flashed image with `Ctrl-R`.
+
+1. Navigate to your new machine's page on the [Viam app](https://app.viam.com).
+   If successful, the status indicator should turn green and show **Live**.
+
+### Configure and test your machine
+
+You can now configure the models you included in your firmware and test them:
+
+1. Navigate to your machine's page in the [Viam app](https://app.viam.com).
+
+1. From the **CONFIGURE** tab, click **JSON** mode.
+   Micro-RDK components and services must be configured in JSON.
+
+1. Add your components and services to the machine configuration.
+   If you are using one of the example modules, find configuration details in the [example modules README](https://github.com/viamrobotics/micro-rdk/blob/main/examples/modular-drivers/README.md#example-modules).
+   For example, if you want to use a free heap sensor, your configuration could look like this:
+
+   ```json
+   {
+     "components": [
+       {
+         "name": "my-free-heap-sensor",
+         "api": "rdk:component:sensor",
+         "model": "free-heap",
+         "attributes": {}
+       }
+     ]
+   }
+   ```
+
+1. Click **Builder** mode and find the configuration card of the component you want to test.
+
+1. Click to open the **Test** section of the card and use the interface to test the component.
+
+1. If you make changes to your modules, rerun the build and flash steps to update the firmware and test the new version on your ESP32.
+
+## Cloud build and over-the-air updates
+
+You can use the Viam cloud build service to build and host your firmware, and use the OTA service to remotely update the firmware on your microcontrollers using the cloud-hosted firmware.
+
+### Build firmware with cloud build
+
+When developing and testing your firmware, it is easiest to build firmware locally.
+When you are ready to deploy your firmware to a fleet of microcontrollers, you can use the Viam cloud build service to simplify the process of building and hosting your firmware by eliminating the need to build locally and then upload the firmware image to a cloud storage bucket.
+
+To build firmware with cloud build, follow these steps:
+
+1. Create a GitHub repository for any [module you created](/operate/get-started/other-hardware/micro-module/) and want to use in your firmware.
+   Upload the contents of your local module directory to the repository.
+
+1. Create a firmware project using the `templates/project` template.
+   _Do not include Wifi credentials or machine cloud credentials in the project_, since you will be publishing the project to a public GitHub repository.
```

Comment:
```suggestion
   _Do not include Wi-Fi credentials or machine cloud credentials in the project_, since you will be publishing the project to a public GitHub repository.
```
[s] Wi-Fi proper name